mixed-content/known-chrome-failures.txt

```diff
@@ -0,0 +1,65 @@
+# IFRAME doesn't trigger events properly in Chrome.
```

Comment:
Why include this file? I agree that it's interesting, but it's better done as bugs filed against Chrome rather than a file in the W3C repository.